index.js

```diff
@@ -67,6 +67,39 @@ function createUser(username, password) {
     });
 }
 
+/**
+ * Configure Git App in SonarQube
+ * @param  {String} projectKey  Sonar project key
+ * @param  {String} projectName Sonar project name
+ * @return {Promise}            Nothing if Git App configured successfully
+ */
+function configureGitApp(projectKey, projectName) {
+    const gitApp = 'Screwdriver Sonar PR Checks';
```

Comment:
better move this `const` out of the function. Also need to document the need for the Github sonar app to be of this name.

Reply:
can have it as a Sonar configuration?